book/src/csi-windows.md

```diff
@@ -0,0 +1,100 @@
+# CSI Windows Suppoort
+
+## Status
+
+Status | Min K8s Version | Min CSI proxy Version | Min Node Driver Registrar Version | latest artifacts
+--|--|--
+Beta | 1.19 | 0.2.0 | 1.3.0 | [binary](gs://k8s-staging-sig-storage/csi-proxy/csi-proxy-master.exe)
+Alpha | 1.18 | 0.1.0 | 1.3.0 |
+
+
+## Overview
+
+CSI drivers (e.g. AzureDisk, GCE PD, etc.) are recommended to be deployed as containers. CSI driver’s node plugin typically runs on every worker node in the cluster (as a DaemonSet). Node plugin containers need to run with elevated privileges to perform storage related operations. However, Windows currently does not support privileged containers. To solve this problem, [CSI Proxy](https://github.com/kubernetes-csi/csi-proxy) - that runs on the Windows host and executes a set of privileged storage operations on Windows nodes on behalf of containers in a CSI Node plugin daemonset. This enables multiple CSI Node plugin authors to execute privileges storage operations on Windows nodes without having to ship a custom privileged operation proxy (until privileged operations get enabled for containers in Windows in the future).
+
+Please note that csi controller level operations/sidecars are not supported on Windows.
+
+## How to use the CSI Proxy for Windows?
+See how to install CSI Proxy in [csi-proxy.md#Deployment]
+
+For CSI driver authors, import CSI proxy client under github.com/kubernetes-csi/csi-proxy/client. There are six client API groups including disk, filesystem, iscsi, smb, system, volume. See [link](https://github.com/kubernetes-csi/csi-proxy/tree/master/client/groups) for details.
+As an example, please check how GCE PD Driver import disk, volume and filesystem API groups [here](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/pkg/mount-manager/safe-mounter_windows.go#L28)
```

Comment:
For links to code with line numbers, it's better to link it at a specific commit because line numbers will change over time.

Reply:
updated